toc.js

```diff
@@ -344,7 +344,16 @@ module.exports = {
         'middleware-doc/doc/instore-app/available-settings/settings',
         'middleware-doc/doc/instore-app/Setup-guide/setup',
         'middleware-doc/doc/instore-app/printer-guide/printer',
-        'middleware-doc/doc/instore-app/multiterminal-settings/multiterminal'
+        'middleware-doc/doc/instore-app/multiterminal-settings/multiterminal',
+        'middleware-doc/doc/instore-app/setup-hobexecr/setup_ecr',
+        {
+          type: 'category',
+          label: 'Installation Guides',
+          items: [
+            'middleware-doc/doc/instore-app/installation-guides/Orderman/orderman-guide',
+            'middleware-doc/doc/instore-app/installation-guides/Sunmi/sunmi-guide'
+          ]
+        }
```

Comment:
Why is this installation guides change also here? It does not belong to this change (and the files are even not existing yet in this PR)